fstar.opam

```diff
@@ -5,7 +5,7 @@ authors: "Nik Swamy <nswamy@microsoft.com>,Jonathan Protzenko <protz@microsoft.c
 homepage: "http://fstar-lang.org"
 license: "Apache-2.0"
 depends: [
-  "ocaml" {>= "4.04.0" & < "4.10.0" }
+  "ocaml" {>= "4.04.1" & < "4.13.0" }
```

Comment:
Do we need the maximum ocaml version here?

Reply:
I guess let's keep it in that PR, but that might be worth a separate PR by me or you later...

Reply:
Yes, that was the reason for the change

Reply:
I didn't realise visitors require >= 4.05.0 though